Manifest.toml

```diff
@@ -0,0 +1,6 @@
+# This file is machine-generated - editing it directly is not advised
+
+julia_version = "1.7.1"
```

Comment:
I wonder should we try out with Julia `v1.6`, because in the `compat` section refers the this version, but you generated these project files with v`1.7`. Please investigate this before get merged.

Reply:
These projekt files work from `v1.6.2`. Should I change `julia_version` to `1.6.2` or regenerate it in the old manifest format? Also, do we need to upload the manifest file? I looked around and there are a lot of Julia repositories without it.

Reply:
Great question. I think that in case of a library such as this, we should probably _not_ add the Manifest.toml to the repository.